File: src/main/java/qageekweek/po/IntroPage.java
package qageekweek.po;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

import qageekweek.DBy;

public class IntroPage extends AbstractPage {

	private static final By SIGNIN_BY = DBy.cssSelector("a[href='/login']","Sign-in in link");
	
	public IntroPage(WebDriver driver) {
		super(driver);
	}
	
	public SignInPage clickOnSignInlnkAndGoToSignInPage() {
		bot.clickOn(SIGNIN_BY);
		return new SignInPage(bot);
	}
	
	
	@Override
	protected void assertInPage() {
		bot.waitForVisible(SIGNIN_BY);
	}

}

File: src/main/kotlin/kqageekweek/po/IntroPage.kt
package kqageekweek.po

import org.openqa.selenium.WebDriver
import qageekweek.DBy
import qageekweek.po.SignInPage

class IntroPage(driver: WebDriver) : AbstractPage(driver) {

    private val SIGNIN_BY = DBy.cssSelector("a[href='/login']", "Sigin in link")

    override fun assertInPage() {
    }

    fun clickOnSignInlnk(): SignInPage {
        bot.clickOn(SIGNIN_BY)
        return SignInPage(bot)
    }


}



